docs/README.Windows.md

```diff
@@ -37,16 +37,16 @@ git clone -b Matrix https://github.com/xbmc/xbmc kodi
 ```
 
 Several different strategies are used to draw your attention to certain pieces of information. In order of how critical the information is, these items are marked as a note, tip, or warning. For example:
- 
-**NOTE:** Linux is user friendly... It's just very particular about who its friends are.  
-**TIP:** Algorithm is what developers call code they do not want to explain.  
+
+**NOTE:** Linux is user friendly... It's just very particular about who its friends are.
```

Comment:
Line-break removal on purpose? :-)

Reply:
Which line break? There was only one on line 40 and it's still there (without whitespaces)

Reply:
The trailing two whitespaces are interpreted as a newline in markdown ([GFM](https://github.github.com/gfm/#hard-line-breaks)). You removed it, hence `TIP` won't be on a separate line).

Reply:
Well, guess we're always learning :) 
Thanks

Reply:
Thanks for updating the PR. Always nice to have a look at Team Kodi's (and your) PRs. I'm always leaning as well. :-)